gnomad_qc/v4/create_release/update_hgdp_tgp_subset_AF.py

```diff
@@ -0,0 +1,700 @@
+"""Script to update the call stats of the HGDP + 1KG subset for v4 release HT."""
+import argparse
+import logging
+from typing import Any, Dict, List, Optional, Tuple
+
+import hail as hl
```

Comment:
I didn't import it because it's not the same order as in the freq_meta.

Reply:
Can you confirm sample numbers to make sure we know the right order, because based on this https://github.com/broadinstitute/gnomad_qc/blob/f6987eff17ed3ad07df03cb9c4469c62804ba7c3/gnomad_qc/v3/create_release/create_release_sites_ht.py#L284 I would have expected it to be in the order of this imported SUBSETS

Reply:
I tested with both, it gave the same results, is `index_globals` doing something to order them in the final freq_meta?